src/test/munit/sender-test-cases.xml

```diff
@@ -0,0 +1,353 @@
+            <email:list-imap config-ref="imap-SenderTests"/>
+            <set-payload value="#[payload[0]]"/>
+            <munit-tools:assert-equals actual="#[payload.payload.body]" expected="#['Email Content']"/>
+            <munit-tools:assert-equals actual="#[payload.attributes.subject]" expected="#['Email Subject']"/>
+            <munit-tools:assert-equals actual="#[payload.attributes.headers.CustomOperationHeader]"
+                                       expected="#['Dummy']"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="sendStreamEmail">
+        <munit:execution>
+            <java:invoke-static class="org.mule.email.mtf.TestSMTPandIMAPServer" method="getContent()" target="stream"/>
+            <email:send config-ref="${sender-config}" fromAddress="juan.desimoni@mulesoft.com"
+                        subject="Email Subject">
+                <email:to-addresses>
+                    <email:to-address value="sebastian.elizalde@mulesoft.com"/>
+                </email:to-addresses>
+                <email:body contentType="text/plain">
+                    <email:content>#[vars.stream]</email:content>
+                </email:body>
+            </email:send>
+        </munit:execution>
+        <munit:validation>
+            <java:invoke-static class="org.mule.email.mtf.TestSMTPandIMAPServer" method="passEmailsToIMAPServer()"/>
+            <email:list-imap config-ref="imap-SenderTests"/>
+            <set-payload value="#[payload[0]]"/>
+            <munit-tools:assert-equals actual="#[payload.payload.body]" expected="#['Email Content']"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="sendEncodedMessage">
+        <munit:execution>
+            <java:invoke-static class="org.mule.email.mtf.TestSMTPandIMAPServer" method="getEncodedBig5Message()"
+                                target="encodedMessage"/>
+            <email:send config-ref="${sender-config}" fromAddress="juan.desimoni@mulesoft.com" subject="Email Subject">
+                <email:to-addresses>
+                    <email:to-address value="sebastian.elizalde@mulesoft.com"/>
+                </email:to-addresses>
+                <email:body contentType="text/plain">
+                    <email:content>#[vars.encodedMessage]</email:content>
+                </email:body>
+            </email:send>
+        </munit:execution>
+        <munit:validation>
+            <java:invoke-static class="org.mule.email.mtf.TestSMTPandIMAPServer" method="passEmailsToIMAPServer()"/>
+            <email:list-imap config-ref="imap-SenderTests"/>
+            <set-payload value="#[payload[0]]"/>
+            <munit-tools:assert-equals actual="#[payload.payload.body]" expected="#[vars.encodedMessage]"/>
+        </munit:validation>
+    </munit:test>
+
+    <!-- TODO complete these tests once the Attachment(SE-10525) and Nobody(MULE-16302) bugs are fixed.-->
```

Comment:
TODO Comments generally has the following pattern: `TODO MULE-XXXXX - asdasdasdsadsa`